src/index.js

```diff
@@ -66,6 +66,12 @@ module.exports = {
 
   util: sharedUtil,
 
+  // Internal shared
+  /* istanbul ignore next */
+  __shared: {
+    errors: require("./common/errors"),
+  },
```

Comment:
Is it okay to share module like this? We can share more modules from `index.js`, `bin-prettier.js` can get even smaller.

Reply:
It's necessary to do something like that. Otherwise there will be multiple instances of the same modules, which might lead to difficult-to-catch errors.